src/components/ChatLog.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+import ChatEntry from './ChatEntry';
+
+const ChatLog = ({ entries, onLikeChange }) => {
+    let isLocal = false;
+
+    return (
+        <div className='chat-log'>
+            {
+                entries.map((entry) => {
+                // Toggle the value of isLocal for each entry
+                isLocal = !isLocal;
+                return (
+                    <ChatEntry
+                        key={entry.id}
+                        sender={entry.sender}
+                        body={entry.body}
+                        timeStamp={entry.timeStamp}
+                        liked={entry.liked}
+                        onLikeChange={onLikeChange}
+                        isLocal={isLocal} // Pass the isLocal value to ChatEntry
```

Comment:
👀 We should pass the `id` for the current message down into the ChatEntry. Even though we're using it for the `key`, if our component needs to use the `id` itself (it does to be able to report which message was clicked, then we need to pass it again.